llvm/lib/Target/AArch64/AArch64SelectionDAGInfo.cpp

```diff
@@ -23,6 +23,23 @@ static cl::opt<bool>
                                 "to lower to librt functions"),
                        cl::init(true));
 
+bool AArch64SelectionDAGInfo::isTargetMemoryOpcode(unsigned Opcode) const {
+  return Opcode >= AArch64ISD::FIRST_MEMORY_OPCODE &&
+         Opcode <= AArch64ISD::LAST_MEMORY_OPCODE;
+}
+
+bool AArch64SelectionDAGInfo::isTargetStrictFPOpcode(unsigned Opcode) const {
+  switch (static_cast<AArch64ISD::NodeType>(Opcode)) {
+  default:
+    return false;
+  case AArch64ISD::STRICT_FCMP:
+  case AArch64ISD::STRICT_FCMPE:
+  case AArch64ISD::SME_ZA_LDR:
+  case AArch64ISD::SME_ZA_STR:
```

Comment:
Can you remove these two from here, as in #120091? If so the Arm/AArch64 parts look Ok to me.

Reply:
I've merged that commit.
Note that these opcodes not included in *MEMORY_OPCODE range -- those are supposed to be used with `getMemIntrinsicNode` (other than that, there isn't much difference).